tests/testthat/test-read_price_data.R

```diff
@@ -0,0 +1,100 @@
+test_that("without specified arguments, read_price_data() throws error", {
```

Comment:
This test and the next actually tests not our code but the functioning of R base,thus the tests are superfluous, I would rather remove them

Reply:
Fair enough, I guess it's hard to argue that my intention was to test R base.

Reply:
that would qualify as hybris